app/grandchallenge/components/__init__.py

```diff
@@ -23,3 +23,21 @@ def check_sagemaker_is_used(app_configs, **kwargs):
         )
 
     return errors
+
+
+@register(deploy=True)
+def check_dummy_provider_is_not_used(app_configs, **kwargs):
+    from django.conf import settings
+
+    errors = []
+
+    if "dummy" in settings.SOCIALACCOUNT_PROVIDERS.keys():
+        errors.append(
+            Error(
+                "The dummy social account provider is configured. This provider should only be used for testing purposes and not in production.",
+                hint="Remove the dummy provider from SOCIALACCOUNT_PROVIDERS in your settings.",
+                id="grandchallenge.components.E002",
```

Comment:
If you do move this update the id as it would no longer be a grandchallenge.components check.